08-interop/test_spec08.py

```diff
@@ -49,6 +50,44 @@ def test_task01(riot_ctrl, log_nodes):
     assert res["stats"]["packet_loss"] < 1
 
 
+@pytest.mark.flaky(reruns=1, reruns_delay=30)
+@pytest.mark.iotlab_creds
+# nodes passed to riot_ctrl fixture
+@pytest.mark.parametrize(
+    'nodes', [pytest.param(['nrf52840dk', 'nrf52840dk'])], indirect=['nodes']
+)
+def test_task03(riot_ctrl):
+    # run `./compile_contiki.sh` relative to this file
+    subprocess.check_call(["./compile_contiki.sh"], cwd=__file__[: __file__.rfind("/")])
+
+    build_path = "build/nrf52840/dk/hello-world.nrf52840"
+    flashfile = f"/tmp/contiki-ng/examples/hello-world/{build_path}"
+
+    gnrc_node, contiki_node = (
+        riot_ctrl(0, GNRC_APP, Shell),
+        riot_ctrl(
+            1,
+            'examples/hello-world',
+            riotctrl.shell.ShellInteraction,
+            extras={"BINFILE": flashfile},
+        ),
+    )
+
+    gnrc_netif, gnrc_addr = lladdr(gnrc_node.ifconfig_list())
+
+    # get the address of the contiki node, ie a substring after "-- "
+    res = contiki_node.cmd("ip-addr")
+    contiki_addr = res[res.find("-- ") + 3 :]
```

Comment:
Let's make flake8 happy :-)

```suggestion
    contiki_addr = res[(res.find("-- ") + 3):]
```

Reply:
Come to think of it: don't you rather want to do it as a regex? E.g.

```suggestion
    match = re.search("-- ...(.+)")
    assert match
    contiki_addr = match[1]
```